apps/ledger-live-mobile/src/components/OperationRow/index.tsx

```diff
@@ -175,20 +188,24 @@ function OperationRow({
           <Text
             variant="body"
             fontWeight="semiBold"
-            color="neutral.c100"
+            color={colors.neutral.c100}
             numberOfLines={1}
           >
             {multipleAccounts ? getAccountName(account) : text}
           </Text>
 
           {isOptimistic ? (
-            <Flex flexDirection="row" alignItems="center">
+            <Flex
+              flexDirection="row"
+              alignItems="center"
+              testID="spinner-container"
+            >
               {spinner}
```

Comment:
if I understand correctly, there was a spinner for all the Optimistic operation. After your PR, there is a spinner only for editable operation. Can you confirm that it is expected ?